vignettes/articles/04-advanced-data-settings.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "04 - Improve stress test by providing further project specific data"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignettes describes in detail how an analysis can be configured further by
+providing further custom data. The configurations are not necessary but, in case
+required information is available, may help you improve the accuracy of your
+results.
+
+## Prerequisites
+In order to keep this vignette concise, we assume that you have successfully
+complete all preparation steps described in the vignettes ["Set up project
+directories"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/01-setup-project-directories.html)
+and [prepare loans
```

Comment:
formatting of the two links seems different?

Reply:
done